src/components/home/Header.jsx

```diff
@@ -3,7 +3,6 @@
 import React from "react"
 import PropTypes from "prop-types"
 import { Link } from "react-router-dom"
-import SinopiaLogo from "styles/sinopia-logo.png"
```

Comment:
should this file be removed from the app?

Reply:
Please.